index.js

```diff
@@ -9,12 +9,18 @@ var blocked = []
 var isBlocked = false
 
 function Plugin(
-	/* config.webpack */ webpackOptions,
-	/* config.webpackServer */ webpackServerOptions,
-	/* config.webpackMiddleware */ webpackMiddlewareOptions,
-	/* config.basePath */ basePath,
-	/* config.files */ files,
-	/* config.frameworks */ frameworks,
+	/* config.webpack */
+	webpackOptions,
```

Comment:
This kind of formatting probably breaks the di injector of karma. So if you need to change that formatting you should use something like this: https://github.com/karma-runner/karma/blob/master/lib/file-list.js#L399-L400 to specify the injects.

```js
function Plugin(webpackOptions) {
}

Plugin.$inject = ['config.webpack']
```